release-doc-in-process/all_vf.TTL

```diff
@@ -30,6 +30,23 @@ vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "A relationship role defining the kind of association one agent can have with another." .
 
+vf:Recipe a               owl:Class ;
+        rdfs:label         "vf:Recipe" ;
+        rdfs:comment       "Recipes contain all the info required to create or improve or move a resource." .
+
+vf:ResourceType a         owl:Class ;
+        rdfs:label         "vf:ResourceType" ;
+        rdfs:comment       "." .
+
+vf:ProcessType a          owl:Class ;
```

Comment:
Since you chose `vf:RecipeFlow` lets go with `vf:RecipeProcess` to represent a process on Recipe layer?

Reply:
Still struggling with this naming.  I need to make sure that this isn't needed outside of recipes - for example for high level processes.  Changed in the meantime, but still open question for me.